package.json

```diff
@@ -67,6 +68,9 @@
     "ts-jest": "^26.2.0",
     "typescript": "^3.9.7"
   },
+  "resolutions": {
+    "chokidar": "3.4.3"
+  },
```

Comment:
So, even after all my efforts trying to install `next` manually, I couldn't get it to work (still got that hunk error) and any `yarn` from scratch would give that error anyway.

Adding this resolution fixes the issue as suggested here: https://github.com/vercel/next.js/pull/21343#issuecomment-777629123 (although I suspect that typed the version number wrong because they're referring to the new one rather than old one)

Reply:
For posterity:

As the above linked issue suggests, yarn 2.4.0 fixed that issue but we have another issue which prevents us from updating. https://github.com/parcel-bundler/parcel/issues/4064#issuecomment-739900240

That seems to have appeared between 2.1.1 (which we are on right now) and 2.2.0.